src/filter.ts

```diff
@@ -18,8 +18,23 @@ limitations under the License.
  * @module filter
  */
 
+import {
+    EventType,
+    RelationType,
+} from "./@types/event";
 import { FilterComponent, IFilterComponent } from "./filter-component";
 import { MatrixEvent } from "./models/event";
+import { UnstableValue } from "./NamespacedValue";
+
+export const UNSTABLE_FILTER_RELATION_SENDERS = new UnstableValue(
+    "relation_senders",
```

Comment:
Wouldn't this be something like `m.relation_senders` ?  Or am I misunderstanding?

Reply:
Nope, the filter fields aren't prefixed with `m.` https://github.com/matrix-org/matrix-doc/blob/gsouquet/threading-via-relations/proposals/3440-threading-via-relations.md